SU-HSDO/suhumsci
# ARCH-300 addressed location dropping to a new line

# READY FOR REVIEW

# Summary
- adjusted styles for time and location (and pipe) for horizontal cards so that the text wraps instead of drops on mobile and/or when location text is long

# Needed By (Date)
- End of sprint (8/2)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that "time | location" does not drop to new line, but rather flows and wraps naturally. /events/conferences in a good page to test

# Affected Projects or Products
- Archaeology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-300

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
